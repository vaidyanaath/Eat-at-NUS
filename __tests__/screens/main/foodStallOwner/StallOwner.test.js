import React from 'react';
import renderer from 'react-test-renderer';
import { act } from 'react-test-renderer';

import StallOwner from '../../../../src/screens/main/foodStallOwner/StallOwner';

describe('<StallOwner />', () => {
    
    it('Renders correctly', () => {
        const mockNavigation = { navigate: () => {} };

        let tree;
        act(() => {
            tree = renderer.create(<StallOwner navigation={mockNavigation} />).toJSON();
        });

        expect(tree).toMatchSnapshot();
        
    });
    
});