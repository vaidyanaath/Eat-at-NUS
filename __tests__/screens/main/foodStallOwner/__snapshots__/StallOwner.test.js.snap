// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StallOwner /> Renders correctly 1`] = `null`;
